### ISSUE TITLE
Securely store user access & refresh tokens

### ISSUE THREAD

**Issue by zeeshanakram3** [Joystream/youtube-synch#36]
After successful Google authentication, the frontend sends the user authorization code to the backend, which is then used to get access & refresh tokens. Both `accessToken` & `refreshToken` are then used for any subsequent retrieval of user/channel state, e.g., polling for new videos. 

Currently, both of these tokens are saved in plaintext in DB, we need to store them securely. AWS natively supports encrypting DynamoDB table, maybe we can look into that

**Comment by bedeho** [comment]
Sounds good!

Is there some sort of possible rate limiting issue we need to think about? say make sure we don't go too intensively on any given token.

**Comment by bedeho** [comment]
Interesting. Will the system we have now be able to pause when limit is hit, and then continue when the next days begins?

**Comment by zeeshanakram3** [comment]
> Will the system we have now be able to pause when limit is hit

Yes, I think, we have a persistent global Quota counter instance that works as proxy b/w incoming requests and the youtube API. It ensures that we only hit the API if we have queries available.
Though haven't tested it yet, probably will do some tests with mocked data + limited quota

> then continue when the next days begin?

Don't think so, but shouldn't be difficult to implement, we would only need to set time for next poll when we will be eligible for queries again.

**Comment by dmtrjsg** [comment]
Low-prio

Since we are managing access to DB actively not important for launch
